specification/programmableconnectivity/Azure.ProgrammableConnectivity/tspconfig.yaml

```diff
@@ -25,12 +25,11 @@ options:
     generate-test: true
     generate-sample: true
   "@azure-tools/typespec-ts":
-    generateMetadata: true
-    generateTest: true
-    azureSdkForJs: true
+    generate-metadata: true
+    generate-test: true
     package-dir: "programmableconnectivity"
-    packageDetails:
-      name: "@azure/programmableconnectivity"
+    package-details:
+      name: "@azure-rest/programmableconnectivity"
```

Comment:
This is added by JS codegen team and considering this is RLC lib so we need to add -rest prefix.